{{ cookiecutter.project_slug }}/Dockerfile

```
@@ -1,4 +1,4 @@
-FROM python:3.7-alpine
+FROM uisautomation/django:2.1-py3.7
```

Taking into account that this is a boilerplate repo and requirements/base.txt uses django>2 I think this should be:

```suggestion
FROM uisautomation/django:py3.7-alpine
```